src/BloomExe/Book/BookStorage.cs

```diff
@@ -866,6 +866,8 @@ private string SanitizeNameForFileSystem(string name)
 				name = name.Replace(c, ' ');
 			}
 			name = name.Trim();
+			if (name.StartsWith("."))
```

Comment:
Best if you do it, give that I already have two cards outsanding :-)

On Wed, Sep 9, 2015 at 3:29 PM, Steve McConnel <notifications@github.com>
wrote:

> In src/BloomExe/Book/BookStorage.cs
> <https://github.com/BloomBooks/BloomDesktop/pull/743#discussion_r39099079>
> :
>
> > @@ -866,6 +866,8 @@ private string SanitizeNameForFileSystem(string name)
> >  				name = name.Replace(c, ' ');
> >  			}
> >  			name = name.Trim();
> > +			if (name.StartsWith("."))
>
> I think your approach would work. Do you want me to abort my pull request
> and have you submit a pull request, or do you want me to revise my pull
> request along those lines? Your approach has the advantage that if the
> title consists only of dangerous characters, a valid filename is produced.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BloomBooks/BloomDesktop/pull/743/files#r39099079>.
>



-- 

John Hatton

Senior Software Engineer/Program Manager
Language Software Development
SIL International